jellyfin.tf

```diff
@@ -170,16 +174,32 @@ resource "aws_key_pair" "jellyfin_keys" {
   public_key = file(".ssh/jellyfin-key.pub")
 }
 
+resource "aws_eip" "jellyfin_eip" {
+  instance = aws_instance.jellyfin_server.id
+  tags = {
+    Name        = "${var.ENVIRONMENT}-jellyfin_eip"
+    Environment = var.ENVIRONMENT
+  }
+  vpc = true
+}
+
 resource "aws_instance" "jellyfin_server" {
   key_name = aws_key_pair.jellyfin_keys.key_name
   ami = data.aws_ami.amazon_linux_2.id
   instance_type = var.EC2_INSTANCE_TYPE
   iam_instance_profile = aws_iam_instance_profile.jellyfin_instance_profile.name
+  associate_public_ip_address = true
   root_block_device {
-    volume_size = var.EBS_VOLUME_SIZE
+    volume_size = var.EBS_ROOT_VOLUME_SIZE
+  }
+  ebs_block_device {
+    device_name = local.EBS_Device 
+    volume_size = var.EBS_MEDIA_VOLUME_SIZE
```

Comment:
Given that we check whether `EBS_MEDIA_VOLUME_SIZE` is empty or not below, this ebs_block_device should be optional based on this as well (unless you're proposing the EBS media volume should be required?)

Reply:
Here's an example that could be used here:
```
dynamic "ebs_block_device" {
    for_each = var.EBS_MEDIA_VOLUME_SIZE != "" ? [var.EBS_MEDIA_VOLUME_SIZE] : []
    content {
      device_name = local.EBS_Device 
      volume_size = var.EBS_MEDIA_VOLUME_SIZE
      volume_type = "st1"
    }
  }
```

Reply:
I was thinking we should make the media volume required, since we could use st1's, which is what I'm using for storage of my media, a little cheaper than a bigger root vol.

Reply:
I don't mind making a separate media volume required and part of the default workflow. It's not any more difficult to manage from a user perspective. Let's remove any conditionals around `EBS_MEDIA_VOLUME_SIZE` then. If someone wants different storage types, they can always customize the media volume.